[2024-06-04 19:35:37,690] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.extrai_dados scheduled__2024-05-27T00:00:00+00:00 [queued]>
[2024-06-04 19:35:37,693] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.extrai_dados scheduled__2024-05-27T00:00:00+00:00 [queued]>
[2024-06-04 19:35:37,693] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-06-04 19:35:37,693] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-06-04 19:35:37,693] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-06-04 19:35:37,705] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): extrai_dados> on 2024-05-27 00:00:00+00:00
[2024-06-04 19:35:37,707] {standard_task_runner.py:52} INFO - Started process 202373 to run task
[2024-06-04 19:35:37,709] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'extrai_dados', 'scheduled__2024-05-27T00:00:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/weather_data.py', '--cfg-path', '/tmp/tmp8r6yaupv', '--error-file', '/tmp/tmp2yuor10e']
[2024-06-04 19:35:37,709] {standard_task_runner.py:80} INFO - Job 21: Subtask extrai_dados
[2024-06-04 19:35:37,736] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.extrai_dados scheduled__2024-05-27T00:00:00+00:00 [running]> on host DESKTOP-BCO05O3.
[2024-06-04 19:35:37,776] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=extrai_dados
AIRFLOW_CTX_EXECUTION_DATE=2024-05-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-27T00:00:00+00:00
[2024-06-04 19:35:38,523] {python.py:173} INFO - Done. Returned value was: None
[2024-06-04 19:35:38,527] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=extrai_dados, execution_date=20240527T000000, start_date=20240604T223537, end_date=20240604T223538
[2024-06-04 19:35:38,564] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-06-04 19:35:38,573] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
